packages/amplify-provider-awscloudformation/src/override-manager/transform-resource.ts

```
@@ -62,5 +61,6 @@ export async function transformResourceWithOverrides(context: $TSContext, resour
     if (spinner) {
       spinner.stop();
     }
+    throw err;
```

this is the main reason why errors were being ignored